reactapp/src/pages/Home.tsx

```diff
@@ -1,26 +1,31 @@
 import Cookies from 'js-cookie';
 import { useEffect, useState } from 'react'
 import {Navigate, useNavigate} from 'react-router-dom'
+import { getChatView } from '../api/api-routes';
 
 const Home = () => {
+ 
   const [auth, setAuth] = useState(false);
   const navigate = useNavigate();
 
   useEffect(() => {
```

Comment:
@Kabewh this will have to be made to only execute once, but this PR will be approved as is for now